pool/example_test.go

```diff
@@ -134,10 +137,17 @@ func ExampleConnectionPool_NewWatcher_noWatchersFeature() {
 	callback := func(event tarantool.WatchEvent) {}
 	watcher, err := connPool.NewWatcher(key, callback, pool.ANY)
 	fmt.Println(watcher)
-	fmt.Println(err)
+	if err != nil {
+		str := err.Error()
+		fmt.Println(strings.Trim(str[:56], " "))
+		fmt.Println(str[56:])
```

Comment:
Please, add a note comment to the confusing code.

Reply:
Updated.